newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -51,6 +59,12 @@ export const OnlineGamePropertiesDialog = ({
   const [playWithMobile, setPlayableWithMobile] = React.useState<boolean>(
     project.isPlayableWithMobile()
   );
+  const [userSlug, setUserSlug] = React.useState<string>(
+    (slug && slug.username) || (profile && profile.username) || ''
+  );
+  const [gameSlug, setGameSlug] = React.useState<string>(
+    (slug && slug.gameSlug) || cleanUpGameSlug(project.getName())
```

Comment:
👍  - this is great, this will suggest a default slug to users

Reply:
Awesome!